flent/runners.py

```diff
@@ -680,8 +680,17 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            self.ctrl_port = args['netperf_control_port']
```

Comment:
Nah, just pushed the change; it's called 'control_port' in the args dict, though, to be consistent with the existing 'control_host' :)